activiti-cloud-services-org/activiti-cloud-services-org-service/src/main/java/org/activiti/cloud/services/organization/service/ModelService.java

```diff
@@ -73,6 +77,10 @@
 
     private final JsonConverter<Model> jsonConverter;
 
+    private final HashMap<String, String> modelIdentifiers = new HashMap();
+
+    private ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
maybe it is better to use the objectMapper already in the context:
private ObjectMapper objectMapper;

and add it in the constructor:
```java
public ClassName(ObjectMapper objectMapper,...){
this.objectMapper = objectMapper;
}
```